src/Components/Task.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react'
+import {
+    Box,
+    Checkbox,
+    ListItem,
+    Typography
+} from '@material-ui/core'
+
+import DeleteIcon from '@material-ui/icons/Delete';
+
+interface Props {
+    task: {
+        title: string,
+        taskId: number
+    },
+    onRemove: Function
+}
+
+const Task: FC<Props> = ({ task, onRemove }: Props) => {
+
```

Comment:
I suggest removing this line.

Reply:
Resolved on dcec6c1c4a6a4ac0c228f8dbef15d6c51c5e9ba7